modules/plot/main/coffee/Utils.coffee

```diff
@@ -1,3 +1,21 @@
+doCollisionDetection = (nodes) ->
+  reductor = (oldDistance, node, currentIndex) ->
+    previousNodes = nodes.slice 0, (currentIndex - oldDistance + 1)
+      .map (previousNode, previousIndex) -> {previousNode, previousIndex}
+    tuple = possibleNewDistance = hx.find previousNodes, ({previousNode, previousIndex}) ->
+      prevBox = previousNode.getBoundingClientRect()
+      currBox = node.getBoundingClientRect()
+      currBox.left < prevBox.right
+    if tuple
+      { previousIndex } = tuple
```

Comment:
We could do this where we call hx.find?

```
{ previousIndex } = hx.find previousNodes, ...
```

In theory, we then can return `hx.max(currentIndex - previousIndex, oldDistance)` and can get rid of this if statement alltogether